packages/go_router/lib/src/path_parser.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
This new file needs unit tests.

(More broadly, it's concerning that there is only one unit test file in this package; hopefully there's a plan for backfilling at least some coverage of the other pieces later on?)

Reply:
I will add some tests. I hadn't add test for this file because the go_router_test.dart would cover the parsing logic as well.